packages/web-react/src/components/CheckboxField/useCheckboxFieldStyleProps.ts

```diff
@@ -22,6 +22,8 @@ export function useCheckboxFieldStyleProps(props: SpiritCheckboxFieldProps): Che
   const checkboxFieldClass = useClassNamePrefix('CheckboxField');
   const checkboxFieldDisabledClass = `${checkboxFieldClass}--disabled`;
   const checkboxFieldItemClass = `${checkboxFieldClass}--item`;
+  const checkboxFieldDangerClass = `${checkboxFieldClass}--danger`;
```

Comment:
there will be other states as well. But we will implement that in the future. Just FYI.